requirements/test-ci.txt

```diff
@@ -1,11 +1,15 @@
 pytest-cov
+codecov; sys_platform == 'win32'
```

Comment:
should this be win32 or !win32?

Reply:
It should be `win32` as this is windows specific as per current `test-ci-windows.txt`